src/CommonUtils.ts

```diff
@@ -88,6 +89,7 @@ export class CommonUtils {
      * @returns parsed xml object.
      */
     public static parseXmlToObject(xmlContent: string): any {
-        return parser.parse(xmlContent, { ignoreAttributes: false, attributeNamePrefix: '', arrayMode: true });
+        const parser: parser.XMLParser = new XMLParser({ ignoreAttributes: false, attributeNamePrefix: ''});
```

Comment:
Please ensure to run `npm run format`
```suggestion
        const parser: parser.XMLParser = new XMLParser({ ignoreAttributes: false, attributeNamePrefix: '' });
```